api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml

```diff
@@ -0,0 +1,27 @@
+model: ernie-lite-8k-0308
+label:
+  en_US: ERNIE-Lite-8K-0308
+model_type: llm
+features:
+  - agent-thought
+model_properties:
+  mode: chat
+  context_size: 8192
+parameter_rules:
+  - name: temperature
+    use_template: temperature
+    min: 0.1
+    max: 1.0
+    default: 0.95
+  - name: top_p
+    use_template: top_p
+    min: 0
+    max: 1.0
+    default: 0.7
+  - name: max_output_tokens
+    use_template: max_output_tokens
```

Comment:
but also, max_output_tokens is not our template, you should alias `max_tokens` to `max_output_tokens` in llm.py